packages/react-tooltip/src/components/Tooltip/useTooltip.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
 import { usePopper } from '@fluentui/react-positioning';
-import { TooltipContext, useFluent } from '@fluentui/react-shared-contexts';
-import { useTheme } from '@fluentui/react-theme-provider';
+import { TooltipContext, useFluent, useTheme } from '@fluentui/react-shared-contexts';
```

Comment:
This is technically a breaking change as well as `useTheme` moved to a different package. Let's hope everybody imports from react-components but would it make sense to add this to breaking changes description?

Reply:
Good catch, I added PR's description 👍